source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, obj, position, _rangeObj=None):
 				log.warning("Couldn't get bounding range for console", exc_info=True)
 				# Fall back to presenting the entire buffer.
 				_rangeObj, collapseToEnd = None, None
-		super(consoleUIATextInfo, self).__init__(obj, position, _rangeObj)
+		super(TextInfo, self).__init__(obj, position, _rangeObj)
```

Comment:
Can't you just use Python 3 `super` here? Same goes for all other modified calls to `super` below.